azext_iot/tests/test_iot_ext_unit.py

```diff
@@ -360,12 +360,32 @@ def generate_device_show(**kvp):
     payload = {
         "authentication": {
             "symmetricKey": {"primaryKey": "123", "secondaryKey": "321"},
+            "type": "sas"
+        },
+        "etag": "abcd",
+        "capabilities": {"iotEdge": True},
+        "deviceId": device_id,
+        "status": "disabled",
+    }
+    for k in kvp:
+        if payload.get(k):
+            payload[k] = kvp[k]
+    return payload
+
+
+def generate_device_show_custom(**kvp):
```

Comment:
What is the difference with the previous function "generate_device_show"

Reply:
updated code as per our discussion and fixed